python/pyhail/dataset.py

```diff
@@ -382,7 +382,56 @@ def annotate_variants_bed(self, input, root, all=False):
     def annotate_variants_expr(self, condition):
         """Annotate variants with expression.
 
-        :param str condition: Annotation expression.
+        **Examples**
+
+        Compute GQ statistics about heterozygotes per variant
+
+        >>> (vds.annotate_variants_expr('va.gqHetStats = '
+        >>>                                'gs.filter(g => g.isHet).map(g => g.gq).stats()'))
+
+        Collect a list of sample IDs with non-ref calls in LOF variants
+
+        >>> (vds.annotate_variants_expr('va.nonRefSamples = gs.filter(g => g.isCalledNonRef).map(g => s.id).collect()'))
+
+        **Details**
```

Comment:
We should "Details", "Implementation Details", and "Notes", variously.  I vote for Notes everywhere.  Thoughts?